examples/vector_env/hello.py

```diff
@@ -28,7 +30,7 @@ class VectorEnvUsage(Enum):
 
                     # Showcase: how to access information from snapshot list in vector env.
                     if env0_dec:
-                        ss0 = env.snapshot_list["vessels"][env0_dec.tick:env0_dec.vessel_idx:"remaining_space"]
+                        ss0 = env.snapshot_list["vessels"][env0_dec.tick : env0_dec.vessel_idx : "remaining_space"]
```

Comment:
Mark.

Compared with line 83 in examples/simulator/data_model.py

Reply:
See [this doc](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#slices) for explanations. I personally buy it.